blobharvest/blob
# Implement event relay for click events

Implements the event relay plan for the CLICKED event (see updated diagram [here](https://app.excalidraw.com/s/96Zjb3zunwk/2eTasThwKpu))

To test:
1. `make run-server`, `make run-client`
2. clear game local storage
3. open game in two windows
4. click 'Go Hunt Shrooma' in both windows
5. click blobalong and move around in one -> should be reflected in other window
6. Note that other actions will not work (spawning, multi-select) - will do these separately.
7. If you want to test for bloblets -> build some in one window -> click score in top left to persist -> refresh both windows -> move bloblet around
